ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/aaa-server-groups.j2

```diff
@@ -1,11 +1,15 @@
 {# eos - AAA Group Servers#}
-{% if aaa_server_groups is defined and aaa_server_groups is not none %}
+{% if aaa_server_groups is arista.avd.defined %}
 !
-{%     for aaa_server_group in aaa_server_groups %}
-aaa group server {{ aaa_server_group['type'] }} {{ aaa_server_group['name'] }}
-{%          for server in aaa_server_group.servers %}
-   server {{ server['server'] }}{% if server['vrf'] is defined and server['vrf'] is not none %} vrf {{ server['vrf'] }}{% endif %}
+{%    for aaa_server_group in aaa_server_groups %}
+{%       if aaa_server_group.type is arista.avd.defined and aaa_server_group.name is arista.avd.defined %}
+aaa group server {{ aaa_server_group.type }} {{ aaa_server_group.name }}
+{%          if aaa_server_group.servers is arista.avd.defined%}
+{%             for server in aaa_server_group.servers %}
+   server {{ server.server }}{% if server.vrf is arista.avd.defined %} vrf {{ server.vrf }}{% endif %}
 
```

Comment:
We should go with a standard multiline if statement or CLI approach used in aaa-accounting template to avoid having a blank line in the middle of the code.